projects/components/src/paginator/paginator.component.ts

```diff
@@ -113,6 +113,9 @@ export class PaginatorComponent implements OnChanges, PaginationProvider {
   @Input()
   public set totalItems(totalItems: number) {
     this._totalItems = totalItems;
+    this.totalRecordsChange.emit(totalItems);
+    this.recordsDisplayedChange.emit(Math.min(this.pageSize, totalItems));
```

Comment:
`Math.min(undefined, 1)` is `NaN`, and `pageSize` is set to `undefined` initially. I'm not sure if that is a state we can get into, though. If it is, I think it would be better to not emit, or to emit `0`.